etc/cvmfs/osgstorage-auth.conf

```diff
@@ -10,42 +10,53 @@ X509_CERT_DIR=
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-houston-stashcache.nrp.internet2.edu:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-sunnyvale-stashcache.t2.ucsd.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ligo.hpc.swin.edu.au:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.bois.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.cinc.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://mghpcc-cache.nationalresearchplatform.org:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osdf-cache.sprace.org.br:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://sdsc-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://xrootd-local.unl.edu:1094/"
+# South America
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osdf-cache.sprace.org.br:8443/"
+# Asia
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://daejeon-kreonet-net.nationalresearchplatform.org:8443/"
+#Europe
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://amst-osdf-xcache01.es.net:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://cf-ac-uk-cache.nationalresearchplatform.org:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ds-914.cr.cnaf.infn.it:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://lond-osdf-xcache01.es.net:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-hawk-cache.cc.lehigh.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.jinr.ru:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.ligo.caltech.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.denv.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.hous.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://dtn-pas.jack.nrp.internet2.edu:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://sc-cache.chtc.wisc.edu:8443/"
+# Oceania
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://singapore.nationalresearchplatform.org:8443/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stash-cache.osg.chtc.io:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stash.farm.particle.cz:8443/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://stashcache.edi.scotgrid.ac.uk:8443/"
+
 # extra authenticated OSDF caches not used by OSG
+# North America
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://sc-cache.chtc.wisc.edu:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-stash-sfu-computecanada-ca.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://osg-hawk-cache.cc.lehigh.edu:8443/"
+# Europe
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://cf-ac-uk-cache.nationalresearchplatform.org:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ds-914.cr.cnaf.infn.it:8443/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https://ligo.hpc.swin.edu.au:8443/"
```

Comment:
Australia should be used by OSG I think.

Reply:
sure